Header header
float32[] x
float32[] y
float32[] z
float32 scan_time
int32[] frames
